test/e2e/framework/pod/wait.go

```diff
@@ -312,6 +313,15 @@ func WaitForMatchPodsCondition(c clientset.Interface, opts metav1.ListOptions, d
 	return fmt.Errorf("gave up waiting for matching pods to be '%s' after %v", desc, timeout)
 }
 
+// GetPodDesc returns a string representing a pod in a consistent human readable format,
+// with pod UID as part of the string.
+// This should match the format defined in kubernetes/pkg/kubelet/util/format.
+func GetPodDesc(podName, podNamespace string, podUID types.UID) string {
```

Comment:
could we do something more verbose, i.e., `GetPodDescription`, or `PodIdentifier`, etc?
`GetPodDesc` sounds crptic :thinking:

Reply:
sure

Reply:
/cc @alejandrox1 
updated